packages/nuclear-deb/nuclear-deb.pacscript

```diff
@@ -0,0 +1,9 @@
+name="nuclear-deb"
+gives="nuclear"
+breaks="${gives} ${gives}-app"
+version="e8fcb6"
```

Comment:
Are the versions based on hashes?

Reply:
Yes

Reply:
There aren't any releases only tags

Reply:
Ugh, I'm not sure that we can have non number starting versions.

Reply:
Hmm as in? since pacstall installed it fine

Reply:
But if you upgrade it, it won't work.

Reply:
Oh.

Reply:
Seems that the latest version is 0.6.17

Reply:
Yes but downloading the deb package doesn't have a tag with that version

Reply:
Then set version to `0.6.17`, then create `_version` and use that one in place of `${version}`

Reply:
So another variable?

Reply:
```name="nuclear-deb"
gives="nuclear"
breaks="${gives} ${gives}-app"
version="0.6.17"
_version="e8fcb6"
description="Streaming music player that finds free music for you "
hash="10b3d9fafbe07ad9feb89747c6cf06e3b2288f021d197f2880042434774e8008"
maintainer="AloneER0 <aloneer0@outlook.com>"
url="https://github.com/nukeop/nuclear/releases/download/${_version}/nuclear-${_version}.deb"
repology=("project: ${gives}")``` like this?